web/app.js

```diff
@@ -424,20 +424,23 @@ const PDFViewerApplication = {
     try {
       const styleSheet = document.styleSheets[0];
       const cssRules = styleSheet?.cssRules || [];
+      let mediaMatcher = "prefers-color-scheme";
+      if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
+        mediaMatcher = "-moz-toolbar-prefers-color-scheme";
+      }
```

Comment:
Nit: I'd probably suggest just setting the value *directly*, rather than first defining it and then immediately overwriting it here:
```javascript
const mediaMatcher =
  typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")
    ? "-moz-toolbar-prefers-color-scheme"
    : "prefers-color-scheme";
```